contents/node/nodejs.md

```diff
@@ -31,11 +31,11 @@ Now that we know what Node is, let us look at some benefits it has to offer.
 
 ### Benefit 1: Easy to get Started
 
-To install Node, simply download the installer from the [official Node website](https://nodejs.org/en/download/) based on your OS and run it. This should install both Node and npm. npm is a tool which will help you to search, install and manage node packages. The use of npm will be explained in the next section.
+To install Node, simply download the installer from the [official Node website](https://nodejs.org/en/download/) based on your OS and run it. This should install both Node and npm. Npm is a tool which will help you to search, install and manage node packages. The use of npm will be explained [later](#benefit-5-easy-dependency-management-with-npm).
 
-Here is an example of how to write a working Node server taken from [codeburst](https://codeburst.io/getting-started-with-node-js-a-beginners-guide-b03e25bca71b).
+The following simple example demonstrates how a Node application imports required modules, creates a server to listen to client's request, and then reads the request and returns a response.
```

Comment:
"The following simple example demonstrates..." sounds slightly awkward for me. I'd have written "The following is a simple example that demonstrates...". Small nitpick, up to you.

Also, I would change "listen to client's request..." to "listen to client requests followed by reading and returning a response for each".